src/tfvc/tfvcerror.ts

```diff
@@ -16,7 +16,18 @@ export class TfvcError {
     tfvcErrorCode: string;
     tfvcCommand: string;
 
-    public constructor(data: ITfvcErrorData) {
+    public constructor(data: ITfvcErrorData | string) {
+        if (typeof data === "string") {
```

Comment:
I wanted a quick way of creating a "missing argument exception". I agree it is confusing. I will create a static method instead.